libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -98,10 +107,168 @@ add_devicelib_obj(libsycl-complex SRC complex_wrapper.cpp DEP ${complex_obj_deps
 add_devicelib_obj(libsycl-complex-fp64 SRC complex_wrapper_fp64.cpp DEP ${complex_obj_deps} )
 add_devicelib_obj(libsycl-cmath SRC cmath_wrapper.cpp DEP ${cmath_obj_deps})
 add_devicelib_obj(libsycl-cmath-fp64 SRC cmath_wrapper_fp64.cpp DEP ${cmath_obj_deps} )
+add_devicelib_obj(libsycl-imf SRC imf_wrapper.cpp DEP ${imf_obj_deps})
+add_devicelib_obj(libsycl-imf-fp64 SRC imf_wrapper_fp64.cpp DEP ${imf_obj_deps})
 
 add_fallback_devicelib(libsycl-fallback-cassert SRC fallback-cassert.cpp DEP ${crt_obj_deps})
 add_fallback_devicelib(libsycl-fallback-cstring SRC fallback-cstring.cpp DEP ${crt_obj_deps})
 add_fallback_devicelib(libsycl-fallback-complex SRC fallback-complex.cpp DEP ${complex_obj_deps})
 add_fallback_devicelib(libsycl-fallback-complex-fp64 SRC fallback-complex-fp64.cpp DEP ${complex_obj_deps} )
 add_fallback_devicelib(libsycl-fallback-cmath SRC fallback-cmath.cpp DEP ${cmath_obj_deps})
 add_fallback_devicelib(libsycl-fallback-cmath-fp64 SRC fallback-cmath-fp64.cpp DEP ${cmath_obj_deps})
+
+# imf fallback is different, we have many separate sources instead of single one including all functions.
+# So, we need to combine all LLVM IR to a complete one and run llvm-spirv for it.
+file(MAKE_DIRECTORY ${obj_binary_dir}/libdevice)
+set(bc_binary_dir ${obj_binary_dir}/libdevice)
+
+set(fallback-imf-src imf_utils/saturatef.cpp
+                     imf_utils/float_convert.cpp
+                     imf_utils/half_convert.cpp
+                     imf/exp_s_la.cpp
```

Comment:
Just curious is it possible to take the section with all Numerics function out of this file in a separate small cmake include file and move it under the ../imf folder? This case Numerics team is going to be responsible to expand this list after all new function additions and no needs for Compiler team assistance required this case.